nc-news/src/components/Dropdown.jsx

```diff
@@ -1,44 +1,40 @@
-import React, {useEffect, useState} from 'react';
-//import {MenuItems} from './MenuItems'
-import {Link} from 'react-router-dom' 
-import './componentsCSS/Dropdown.css'
-import * as api from '../api'
+import React, { useEffect, useState } from "react";
+import { Link } from "react-router-dom";
+import "./componentsCSS/Dropdown.css";
+import * as api from "../api";
 
-function Dropdown(){
-    const [click, setClick] = useState(false)
-    const [categories, setCategory] = useState([])
+function Dropdown() {
+  const [click, setClick] = useState(false);
+  const [categories, setCategory] = useState([]);
 
-    let MenuItems = [
-      {  topic: 'All',
-          path: '/articles',
-          name: 'dropdown-link'
-        }]
+  let MenuItems = [{ topic: "All", path: "/articles", name: "dropdown-link" }];
 
-    const handleClick=()=>setClick(!click)
+  const handleClick = () => setClick(true);
 
-    useEffect(()=>{
-             api.getTopics().then((categoriesAPI)=>{
-              setCategory(categoriesAPI)
-             })
-          }, [])
-   
-      categories.forEach(top =>{
-        let topic = top.slug
-        
-        MenuItems.push({
-            topic: `${topic}`,
-            path: `/topics/${topic}`,
-            name: 'dropdown-link'
-          })
-      })
+  useEffect(() => {
+    api.getTopics().then((categoriesAPI) => {
+      setCategory(categoriesAPI);
+    });
+  }, []);
 
-    return(
-        <>
-        <ul 
+  categories.forEach((top) => {
+    let topic = top.slug;
+    topic.toLowerCase();
+    topic[0].toUpperCase();

```

Comment:
strings are not mutable so calling `toLowerCase()` or `toUpperCase()` by themselves won't change the `topic` string

you'd need to assign the output to a variable and use that

Reply:
this bit of code is still not doing anything as strings are immutable